django_project/ford3/models/campus.py

```diff
@@ -126,13 +126,17 @@ def events(self):
 
     @property
     def qualifications(self):
-        return list(self.qualification_set.all().values(
+        qualifications = self.qualification_set.filter(
+            campus_id=self.id,
+            deleted=False
```

Comment:
why not use the db manager?

Reply:
So I wanted to but I am getting an "Unable to import Qualification" error - I assumed this was from a circular reference but I just couldn't find the reason. After it started taking time I reverted to this - not ideal but it works and we can use the manager in other spots we need.